addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1126,6 +1126,12 @@ export class Orderline extends PosModel {
     isPartOfCombo() {
         return Boolean(this.comboParent || this.comboLines?.length);
     }
+    getComboTotalPrice() {
+        const allLines = this.getAllLinesInCombo();
+        return allLines.reduce((total, line) => {
+            return (total += line.get_price_with_tax());
```

Comment:
don't mutate the accumulator
[suggestion](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#try_it)
```js
array1.reduce(
  (accumulator, currentValue) => accumulator + currentValue,
  initialValue,
);
```